server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -202,42 +202,27 @@ const saveClinicalDetails = (request: Request, response: Response, baseLog: Init
 }
 
 clinicalDetailsRoute.post('/saveClinicalDetails', (request: Request, response: Response) => {
-    const isolationAddressId = request.body.clinicalDetails?.isolationAddressId;
     const logData = {
         workflow: `saving clinical details tab`,
         investigation: response.locals.epidemiologynumber,
         user: response.locals.user.id
     };
     const isolationAddress = formatToNullable(request.body.clinicalDetails?.isolationAddress);
-    if (!isolationAddressId) {
+    if (isolationAddress !== null) {
         const requestAddress: InsertAndGetAddressIdInput = formatToInsertAndGetAddressIdInput(isolationAddress);
         const createAddressLogger = logger.setup({...logData, workflow: `${logData.workflow}: create isolation address`})
-
         const parameters = {input: requestAddress};
         createAddressLogger.info(launchingDBRequestLog(parameters), Severity.LOW);
-
-        graphqlRequest(CREATE_ADDRESS, response.locals, parameters)
-        .then((result: CreateAddressResponse) => {
+        graphqlRequest(CREATE_ADDRESS, response.locals, {
+            input: requestAddress
```

Comment:
you can still use the 'parameters' variable

Reply:
👍🏼